serverless/cmd/client/client.go

```diff
@@ -86,9 +82,11 @@ func main() {
 	}
 
 	// Derive logID from log public key
-	v, _ := note.NewVerifier(pubKey)
-	key_hash := v.KeyHash()
-	logID := fmt.Sprint(key_hash)
+	v, err := note.NewVerifier(pubKey)
+	if err != nil {
+		glog.Exitf("Failed to instantiate Verifier : %q", err)
+	}
+	logID := fmt.Sprint(v.KeyHash())
```

Comment:
tiny nit (sorry i missed this last time): should probably be explicit here, so either `fmt.Sprintf("%d", v.KeyHash())` or `strconv.FormatInt(v.KeyHash(), ...)`

Reply:
🌮

Reply:
Done